### PR TITLE
Deprecated memberHost/Port; added instead mappedHost/Port

### PR DESCRIPTION
**Pull request by artem-v**
Fixes https://github.com/scalecube/scalecube-cluster/issues/285

### CODE REVIEW

File: cluster-api/src/main/java/io/scalecube/cluster/ClusterConfig.java [hunk 1]
```diff
@@ -152,28 +152,67 @@ public ClusterConfig metadataDecoder(MetadataDecoder metadataDecoder) {
...
+
+  /**
+   * Setter for mappedHost. {@code mappedHost} is a config property for container environments, it's
+   * being set for advertising to scalecube cluster some connectable hostname which maps to
+   * scalecube transport's hostname on which scalecube transport is listening.
+   *
+   * @param mappedHost mapped host
+   * @return new {@code ClusterConfig} instance
+   */
+  public ClusterConfig mappedHost(String mappedHost) {
```
Comment (ronenhamias) [inline cluster-api/src/main/java/io/scalecube/cluster/ClusterConfig.java]: maybe the name is better to be exportedHostName / externalHost / containerHost

mapped name is a term taken from what world?

for example in dockers its export
in routers its mapping?

**Comment by ronenhamias** [comment]
add to examples a simple example to showcase the usage

File: cluster-api/src/main/java/io/scalecube/cluster/ClusterConfig.java (hunk 1 above)
Comment (artem-v) [reply cluster-api/src/main/java/io/scalecube/cluster/ClusterConfig.java]: Are you good with `containerHost` and `containerPort`?

Comment (segabriel) [reply cluster-api/src/main/java/io/scalecube/cluster/ClusterConfig.java]: For me, `mapped*` was better

**Comment by artem-v** [comment]
> add to examples a simple example to showcase the usage

I have created https://github.com/scalecube/scalecube-cluster/issues/288. It's too much to do for it in this PR.

**Pull request merged**